opentimelineview/timeline_widget.py

```diff
@@ -145,24 +148,55 @@ def _set_labels_timecode(self):
             )
         )
 
+    def _set_labels_effects(self):
+        if isinstance(self.item, otio.core.Item):
+            if self.item.effects:
+                self.source_fx_label.setHtml(
+                    "<div style='background:rgba(255,255,255,100%);'>FX</div>"
+                )
+            else:
+                self.source_fx_label.setPlainText('')
+
     def _set_labels(self):
         self._set_labels_rational_time()
         self.source_name_label.setText('PLACEHOLDER')
         self._position_labels()
 
     def _set_tooltip(self):
-        self.setToolTip(self.item.name)
+        toolTipStr = self.item.name
+
+        if isinstance(self.item, otio.schema.Clip) or \
+           isinstance(self.item, otio.schema.Stack):
+            duration = "%0.2f" % otio.opentime.to_seconds(
```

Comment:
We've been using the `.format()` syntax to stay python 3 compatible, but it works almost the same:

`"{0:.2f}".format(otio.opentime.to_seconds(...)`

Should do it.